src/components/UserProfile/PropertyCarousel.js

```diff
@@ -0,0 +1,47 @@
+import Paper from "@mui/material/Paper";
+import Typography from '@mui/material/Typography';
+import Carousel from 'react-material-ui-carousel';
+import Stack from "@mui/material/Stack";
+
+/**
+ * Creates a carousel for the given properties.
+ * TODO: properties is currently just an array of strings, display real property cards here.
+ */
+export default function PropertyCarousel({ properties, title }) {
+	let interestedProp = properties;
+
+	// from https://stackoverflow.com/a/44069560
+	function groupArr(data, n) {
+		let group = [];
+		for (let i = 0, j = 0; i < data.length; i++) {
+			if (i >= n && i % n === 0)
+				j++;
+			group[j] = group[j] || [];
+			group[j].push(data[i])
+		}
+		return group;
+	}
+
+	// TODO: make reactive by changing groupNumber based on screen size
+	// Unsure how to check breakpoint programatically
+	let groupsOf3 = groupArr(interestedProp, 3);
+
+	return (
+		<Paper variant="outlined" margin={3} padding={3}>
```

Comment:
```suggestion
    <Paper 
      variant="outlined" 
      sx={{
                    mt: { xs: 3, md: 6 },
                    mb: { xs: 3 },
                    p: { xs: 2, md: 3 },
                }}>
```

my paper margin and padding looks like this. This is also in the signup flow. To keep it consistent you can update it to these values. don't forget the paper components in the other files too :)